js_modules/dagit/packages/core/src/assets/AssetTable.tsx

```diff
@@ -138,105 +156,142 @@ const AssetEntryRow: React.FC<{
-        <Description>
-          {asset?.definition &&
-            asset.definition.description &&
-            markdownToPlaintext(asset.definition.description).split('\n')[0]}
-        </Description>
-      </td>
-      <td>
-        {asset?.definition && (
-          <Box flex={{direction: 'column', gap: 2}}>
-            <RepositoryLink
-              showIcon
-              showRefresh={false}
-              repoAddress={{
-                name: asset.definition.repository.name,
-                location: asset.definition.repository.location.name,
-              }}
-            />
-          </Box>
-        )}
-      </td>
-      <td>
-        {asset ? (
-          <Box flex={{gap: 8, alignItems: 'center'}}>
-            {asset.definition?.opNames.length ? (
-              <Link
-                to={instanceAssetsExplorerPathToURL({
-                  opsQuery: `++"${tokenForAssetKey({path})}"++`,
-                  opNames: [tokenForAssetKey({path})],
-                })}
-              >
-                <Button>View in Asset Graph</Button>
-              </Link>
-            ) : (
-              <Button disabled={true}>View in Asset Graph</Button>
-            )}
-            <Popover
-              position="bottom-right"
-              content={
-                <Menu>
-                  <MenuLink
-                    text="View details…"
-                    to={`/instance/assets/${path.join('/')}`}
-                    icon="view_list"
-                  />
-                  <MenuItem
-                    text="Wipe Asset…"
-                    icon="delete"
-                    disabled={!canWipe}
-                    intent="danger"
-                    onClick={() => canWipe && onWipe(assets)}
+    const onChange = (e: React.FormEvent<HTMLInputElement>) => {
+      if (e.target instanceof HTMLInputElement) {
+        const {checked} = e.target;
+        const shiftKey =
+          e.nativeEvent instanceof MouseEvent && e.nativeEvent.getModifierState('Shift');
+        onToggleChecked({checked, shiftKey});
+      }
+    };
+
+    const liveData = asset && liveDataByNode[toGraphId(asset.key)];
+    const stepKey = asset && asset.definition?.opNames[0];
+
+    return (
+      <tr>
+        <td style={{paddingRight: '4px'}}>
+          <Checkbox checked={isSelected} onChange={onChange} />
+        </td>
+        <td>
+          <AssetLink
+            path={path}
+            url={linkUrl}
+            isGroup={isGroup}
+            icon={isGroup ? 'folder' : asset?.definition ? 'asset' : 'materialization'}
+          />
+          <Description>
+            {asset?.definition &&
+              asset.definition.description &&
+              markdownToPlaintext(asset.definition.description).split('\n')[0]}
+          </Description>
+        </td>
+        <td>
+          {asset?.definition && (
+            <Box flex={{direction: 'column', gap: 2}}>
```

Comment:
`gap` necessary?

Reply:
Removed! Also removed it from the other place I copy pasted this from 😅